support-frontend/assets/components/thankYou/supportReminder/supportReminderItems.tsx

```diff
@@ -222,18 +216,11 @@ export function SupportReminderCTAandPrivacy({
 			})
 			.catch(catchPromiseHandler('Error creating reminder sign up'));
 	};
-
 	const onSubmit = () => {
 		setReminder();
 		trackComponentClick(OPHAN_COMPONENT_ID_SET_REMINDER);
-		dispatch(
-			setThankYouSupportReminder({
-				...supportReminderState,
-				hasBeenCompleted: true,
-			}),
-		);
+		onClick ? onClick() : undefined;
```

Comment:
You can use optional chaining here - `onClick?.()`

Reply:
Ah yes, neat! Done